crates/ruff_linter/resources/test/fixtures/flake8_simplify/SIM115.py

```diff
@@ -59,6 +60,25 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.file.close()
 
 
+# OK
+class ExampleClassTests(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.enterClassContext(open("filename"))
+
+
+# OK
+class ExampleAsyncTests(IsolatedAsyncioTestCase):
+    async def test_something(self):
+        await self.enterAsyncContext(open("filename"))
+
+
+# OK
+class ExampleTests(TestCase):
+    def setUp(self):
+        self.enterContext(open("filename"))
```

Comment:
would you mind moving these (and the new import) to the bottom of the fixture file, then regenerating the snapshot? Adding them to the middle results in a _lot_ of line numbers changing in the snapshot, which makes it very hard to review, unfortunately :-(